### ISSUE TITLE
ArgumentException (OculusHand) on import of the Foundation package

### ISSUE THREAD

**Issue by davidkline-ms** [microsoft/MixedRealityToolkit-Unity#8435]
When importing the Foundation package (.unitypackage or via UPM), the following error is encountered.

```
ArgumentException: The Object you want to instantiate is null.
UnityEngine.Object.CheckNullArgument (System.Object arg, System.String message) (at <ee47be73f7ef409ca5e5ce4b121745b7>:0)
UnityEngine.Object.Instantiate[T] (T original) (at <ee47be73f7ef409ca5e5ce4b121745b7>:0)
Microsoft.MixedReality.Toolkit.XRSDK.Oculus.MRTKOculusConfig.get_Instance () (at Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs:46)
Microsoft.MixedReality.Toolkit.XRSDK.Oculus.OculusHand..ctor (Microsoft.MixedReality.Toolkit.TrackingState trackingState, Microsoft.MixedReality.Toolkit.Utilities.Handedness controllerHandedness, Microsoft.MixedReality.Toolkit.Input.IMixedRealityInputSource inputSource, Microsoft.MixedReality.Toolkit.Input.MixedRealityInteractionMapping[] interactions) (at Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Input/Controllers/OculusHand.cs:85)
System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) (at <fb001e01371b4adca20013e0ac763896>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) (at <fb001e01371b4adca20013e0ac763896>:0)
System.Reflection.MonoCMethod.DoInvoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <fb001e01371b4adca20013e0ac763896>:0)
System.Reflection.MonoCMethod.Invoke (System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <fb001e01371b4adca20013e0ac763896>:0)
System.RuntimeType.CreateInstanceImpl (System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Threading.StackCrawlMark& stackMark) (at <fb001e01371b4adca20013e0ac763896>:0)
System.Activator.CreateInstance (System.Type type, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes) (at <fb001e01371b4adca20013e0ac763896>:0)
System.Activator.CreateInstance (System.Type type, System.Object[] args) (at <fb001e01371b4adca20013e0ac763896>:0)
Microsoft.MixedReality.Toolkit.Input.MixedRealityControllerMapping.GetDefaultInteractionMappings () (at Assets/MRTK/Core/Definitions/Devices/MixedRealityControllerMapping.cs:220)
Microsoft.MixedReality.Toolkit.Input.MixedRealityControllerMapping.UpdateInteractionSettingsFromDefault () (at Assets/MRTK/Core/Definitions/Devices/MixedRealityControllerMapping.cs:145)
Microsoft.MixedReality.Toolkit.Input.MixedRealityControllerMappingProfile.AddMappings () (at Assets/MRTK/Core/Definitions/Devices/MixedRealityControllerMappingProfile.cs:182)
Microsoft.MixedReality.Toolkit.Input.MixedRealityControllerMappingProfile.Awake () (at Assets/MRTK/Core/Definitions/Devices/MixedRealityControllerMappingProfile.cs:133)
```

This is occurring on both Unity 2018.4 and 2019.4.

**Comment by RogPodge** [comment]
yeah this should get fixed with https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8386

**Comment by keveleigh** [comment]
@RogPodge Not an exception, but I'm also seeing 
![image](https://user-images.githubusercontent.com/3580640/92023783-196d2900-ed12-11ea-98c6-9df6269c4410.png)

Any idea if those can be cleaned up?

**Comment by elbuhofantasma** [comment]
This issue also occurs on this commit: a97278881

**Comment by RogPodge** [comment]
@elbuhofantasma is this issue fixed at all with the latest code from #8386

**Comment by RogPodge** [comment]
Closing here since CI is passing \o/
